docs/resources/kibana_import_saved_objects.md

```diff
@@ -0,0 +1,99 @@
+---
+subcategory: "Kibana"
+layout: ""
+page_title: "Elasticstack: elasticstack_kibana_import_saved_objects Resource"
+description: |-
+  Create sets of Kibana saved objects from a file created by the export API.
+---
+
+# Resource: elasticstack_kibana_import_saved_objects
+
+Create sets of Kibana saved objects from a file created by the export API. See https://www.elastic.co/guide/en/kibana/current/saved-objects-api-import.html
+
+## Example Usage
+
+```terraform
+provider "elasticstack" {
+  kibana {}
+}
+
+resource "elasticstack_kibana_import_saved_objects" "settings" {
+  overwrite     = true
+  file_contents = <<-EOT
```

Comment:
Does it make sense to use `jsonencode` instead of heredoc - https://developer.hashicorp.com/terraform/language/expressions/strings#generating-json-or-yaml?

Reply:
Actually, maybe it makes sense to use heredoc which may be a more convenient tool for inserting the file generated by the API.